spec/models/league_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "League model", type: :model do
+
+    it "should get created with correct data" do
+        league = League.new(name: "Weitwurfmeisterschaften",
```

Comment:
How about defining the object in a before/let block and then making different tests for all the expect statements? So that we know what field exactly is actually not working instead of having this one big test fail.

Reply:
This test mostly seems to check whether the constructor behaves as expected. The factory for league models that is also part of this PR isn't used. Maybe https://github.com/hpi-swt2/sport-portal/blob/dev/spec/models/match_spec.rb could be an inspiration?